velox/expression/tests/ExpressionRunner.cpp

```diff
@@ -27,40 +29,113 @@
 
 namespace facebook::velox::test {
 
-using namespace facebook::velox;
+namespace {
+std::vector<core::TypedExprPtr> parseSql(
+    const std::string& sql,
+    const TypePtr& inputType,
+    memory::MemoryPool* pool) {
+  auto exprs = parse::parseMultipleExpressions(sql, {});
+
+  std::vector<core::TypedExprPtr> typedExprs;
+  typedExprs.reserve(exprs.size());
+  for (const auto& expr : exprs) {
+    typedExprs.push_back(core::Expressions::inferTypes(expr, inputType, pool));
+  }
+  return typedExprs;
+}
+
+RowVectorPtr createRowVector(
```

Comment:
nit: can you add a small description of what this utility function is used for ( like changing column name to something user friendly).